File: web/backend/src/models/restaurantDish.js
module.exports = (sequelize, DataTypes) => {
  const RestaurantDish = sequelize.define('RestaurantDish', {
    restaurant_id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
    },
    dish_id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
    },
    name: DataTypes.STRING,
  }, {
    tableName: 'restaurant_dishes',
    timestamps: false,
  });
  return RestaurantDish;
};

File: old_version/web/backend/src/models/restaurant_dish.py
from sqlalchemy import Column, Integer, String, ForeignKey
from ..config.database import Base

class RestaurantDish(Base):
    __tablename__ = "restaurant_dishes"

    restaurant_id = Column(Integer, ForeignKey("restaurants.id"), primary_key=True)
    dish_id = Column(Integer, ForeignKey("dishes.id"), primary_key=True)
    name = Column(String(255))